### PR TITLE
New books-of-bible.json

### PR DESCRIPTION
**Pull request by jophab**
I am sorry that i again made the mistake of creating PR from master.

**What does your Instant Answer do?**
shows books in bible
**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.biblebelievers.com/believers-org/kjv-stats.html
**Why did you choose this data source?**
Its good
**Are there any other alternative (better) data sources?**
yes
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Mostly christians
**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
nope
**Are you having any problems? Do you need our help with anything?**
no
**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
image
image

-----
IA Page: https://duck.co/ia/view/bible_books_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@jophab I would close the PR and start over with a branch otherwise you may run into issue if you try to work on anything else.

Also, please run this through JSONLint or similar to correct the indentation

**Comment by jophab** [comment]
@moollaza Dont worry I will wait till this one is merged.
Or will be careful in my other works

File: share/goodie/cheat_sheets/json/books-of-bible.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    "template_type": "reference",
+    "section_order": [
+        "Old Testament",
+        "New Testament"
+    ],
+    "sections": {
+        "Old Testament": [
+            {
+                "key": "Genesis",
+                "val": "No.of Chapters : 50 , No.of Verses : 1,533"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/books-of-bible.json]: let's flip these and simplify them:

>**Genesis**
> 1,533 verses, 50 chapters

Comment (moollaza) [inline share/goodie/cheat_sheets/json/books-of-bible.json]: Also, these sections are too big. Are these in order? I think we'll need to break it up by book numbers:

> **Old Testament (Book 1 - 10)**
>* **Genesis (Book 1)**  
>  1,533 verses, 50 chapters

> **Old Testament (Book 11 - 20)**
>* **Genesis (Book 1)**  
>  1,533 verses, 50 chapters

File: share/goodie/cheat_sheets/json/books-of-bible.json [hunk 2]
```diff
@@ -0,0 +1,294 @@
+{
+    "id": "books_of_bible_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/books-of-bible.json]: I think `bible-books.json` and `bible_books_cheat_sheet` would be a better?

File: share/goodie/cheat_sheets/json/books-of-bible.json [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+    "name": "Books of The Holy Bible",
+    "description": "List of books in The Holy Bible along with their Number of Chapters and Number of Verses",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/books-of-bible.json]: no need to capitalize `Number of Chapters and Number of Verses`

File: share/goodie/cheat_sheets/json/books-of-bible.json (hunk 1 above)
Comment (jophab) [inline share/goodie/cheat_sheets/json/books-of-bible.json]: yes they are in order.
I will split them . :+1:

**Comment by jophab** [comment]
@moollaza changed file name and id, modified desc.
I will do the splitting shortly.. :+1: (exams going on)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/bible_books_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/bible_books_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/books-of-bible.json (hunk 1 above)
Comment (jophab) [inline share/goodie/cheat_sheets/json/books-of-bible.json]: @moollaza Is the flipping necessary?

**Comment by jophab** [comment]
@moollaza  Is the flipping of val field contents necessary?

**Comment by jophab** [comment]
@moollaza Everything you suggested is corrected. :+1: 
Please check.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/bible_books_cheat_sheet_mobile.png)

**Comment by jophab** [comment]
@moollaza

**Comment by jophab** [comment]
@moollaza any more changes?

**Comment by moollaza** [comment]
@jophab Thanks! This LGTM :+1:

**Comment by moollaza** [comment]
@jophab congrats! It's live: https://duckduckgo.com/?q=Bible%20Books%20Cheat%20Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it.

**Pull request merged**